docs/docusaurus/docs/reference/learn/data_quality_use_cases/unstructured_data/unstructured_data_process_files.py

```diff
@@ -0,0 +1,68 @@
+# <snippet name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/unstructured_data/unstructured_data_process_files.py - load the dataset">
+from datasets import load_dataset  # Load PDF OCR dataset from Hugging Face
+
+ds = load_dataset("broadfield-dev/pdf-ocr-dataset", split="train[:5]")
+# </snippet>
+
+# <snippet name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/unstructured_data/unstructured_data_process_files.py - iterate through the data">
+import pytesseract
```

Comment:
This import used to have a comment which I thought was helpful

```suggestion
import pytesseract # OCR engine
```